src/app/page/UserProfile/UserProfile.js

```diff
@@ -75,9 +76,39 @@ const UserProfile = ({ history }) => {
     setProfile(_.cloneDeep(firebaseProfile));
     setView("view");
   }
-  function saveProfileAction(e) {
+  async function saveProfileAction(e) {
     e.preventDefault();
-    firebase.updateProfile(profile);
+    try {
+      !profile.address
+        ? console.log(null)
+        : await addressLookUp(profile.address).then((data) =>
```

Comment:
wait...why do we have a await and also a .then()...

Reply:
I'll remove the async / await